InSituLib/VtkMappedMeshSource.h

```diff
@@ -84,9 +84,10 @@ class VtkMappedMeshSource : public vtkUnstructuredGridAlgorithm
 	/// \param properties MeshLib::Properties object
 	/// \param prop_name The name of the property vector to be mapped from
 	/// vtk-mesh to ogs-mesh
-	template<typename T> bool addProperty(vtkUnstructuredGrid &output,
-	                                      MeshLib::Properties const& properties,
-	                                      std::string const& prop_name) const
+	template <typename T>
+	bool addProperty(vtkUnstructuredGrid& /*output*/,
```

Comment:
Removed; but there is more in `VtkMappedMeshSource.cpp` on `output` variable, which I didn't touch. Guess it is for another PR...